prijateli_tree/app/routers/administration.py

```diff
@@ -120,3 +121,53 @@ def dashboard(
             "transactions": denir_transactions,
         },
     )
+
+
+@router.post("/game")
+def create_game(
+    request: Request,
+    game_type: Annotated[str, Form()],
+    rounds: Annotated[int, Form()],
+    practice: Annotated[bool, Form()],
+    pos_one: Annotated[int, Form()],
+    pos_two: Annotated[int, Form()],
+    pos_three: Annotated[int, Form()],
+    pos_four: Annotated[int, Form()],
+    pos_five: Annotated[int, Form()],
+    pos_six: Annotated[int, Form()],
+    user=Depends(login_manager.optional),
+    db: Session = Depends(get_db),
+):
+    if user is None:
+        return RedirectResponse("login", status_code=HTTPStatus.FOUND)
+
+    game = Game(
+        created_by=user.id,
+        game_type_id=game_type,
+        rounds=rounds,
+        practice=practice,
+    )
+
+    db.add(game)
+    db.commit()
+    db.refresh(game)
+
+    # TODO: Who has their name hidden?
```

Comment:
@FedericoDM, how is this determined?

Reply:
@michplunkett Use the admin panel to start "Sessions". During a session, the 6 players play many games of different types which are randomized (e.g. integrated / segregated and with/without names). It would make sense to use this code within a session to create the games.

Reply:
The way I understood from the get-go is that each game is a discrete instance. So one game is integrated, another is segregated, etc. That's the way it is currently setup.

Reply:
As far as where this will be housed, it will be on the admin panel button in this screen:
![image](https://github.com/prijatelilab/PrijateliTree/assets/5885605/4baf99dc-a3ed-4965-84d8-75cf50b664e7)

Reply:
Here's the select:
<img width="498" alt="Screenshot 2023-11-13 at 3 23 13 PM" src="https://github.com/prijatelilab/PrijateliTree/assets/5885605/26825a0b-5084-4061-a224-bb48103ec48a">

Reply:
Since this isn't a React app or something, this form will have to be a separate page.